tests/Get-DbaDbDetachedFileInfo.Tests.ps1

```diff
@@ -39,7 +40,7 @@ Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
             $results.name | Should Be $dbname
         }
         It "Should be 2016" {
```

Comment:
if this doesn't base itself around having a fixed version and checking it, this "IT" needs to be adjusted as well